contracts/FiatGateway.sol

```diff
@@ -161,22 +161,22 @@ contract FiatGateway is Upgradeable, ERC1155HolderUpgradeable {
         address buyer,
         address seller,
         uint nftId,
+        uint quantity,
```

Comment:
@neanvo Is everything updated to this change? Or should we make a backward-compatible change instead?